src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,61 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+import { getMovie } from '../../api';
+
+type Props = {
+  addMovie: (newMovie: Movie) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [searchText, setSearchText] = useState<string>('');
+  const [errorThroughSearch, setErrorThroughSearch] = useState(false);
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [loader, setLoader] = useState(false);
+
+  const textOfSubmitButton = movie
+    ? 'Search again'
+    : 'Find a movie';
+
+  function handleSetSearchText(
+    event: React.ChangeEvent<HTMLInputElement>,
+  ) {
+    setSearchText(event.target.value);
+    setErrorThroughSearch(false);
+  }
+
+  const adddMovieToList = () => {
```

Comment:
The function name 'adddMovieToList' seems to be a typo. It should be 'addMovieToList'.